pyproject.toml

```diff
@@ -8,11 +8,17 @@ authors = ["xal9wiii4ik <xal9wiii4ik@yandex.ru>"]
 python = "^3.8"
 requests = "^2.26.0"
 Flask = "^2.0.2"
-python-dotenv = "^0.19.2"
 Flask-SQLAlchemy = "^2.5.1"
 psycopg2-binary = "^2.9.2"
 gunicorn = "^20.1.0"
-pre-commit = "^2.16.0"
+Flask-Migrate = "^3.1.0"
+Flask-Script = "^2.0.6"
+Flask-Pydantic = "^0.9.0"
+PyJWT = "^2.3.0"
+Flask-Cors = "^3.0.10"
+boto3 = "^1.20.41"
+fleep = "^1.0.1"
```

Comment:
why you need this?

Reply:
done

Reply:
then you should include it in the future PR